up42/project.py

```diff
@@ -143,25 +149,49 @@ def get_jobs(
             return_json: If true, returns the job info jsons instead of JobCollection.
             test_jobs: Return test jobs or test queries.
             real_jobs: Return real jobs.
+            limit: Optional, only return n first assets by sorting criteria and order.
+                Optimal to select if your workspace contains many assets.
+            sortby: The sorting criteria, one of "createdAt", "name", "id", "mode", "status", "startedAt", "finishedAt".
+            descending: The sorting order, True for descending (default), False for ascending.
 
         Returns:
             All job objects in a JobCollection, or alternatively the jobs info as json.
         """
+        allowed_sorting_criteria = [
+            "createdAt",
+            "name",
+            "id",
+            "mode",
+            "status",
+            "startedAt",
+            "finishedAt",
+        ]
+        if sortby not in allowed_sorting_criteria:
+            raise ValueError(
+                f"sortby parameter must be one of {allowed_sorting_criteria}!"
+            )
+        sort = f"{sortby},{'desc' if descending else 'asc'}"
+
         page = 0
-        url = f"{self.auth._endpoint()}/projects/{self.project_id}/jobs?page={page}"
+        url = f"{self.auth._endpoint()}/projects/{self.project_id}/jobs?page={page}&sort={sort}"
         response_json = self.auth._request(request_type="GET", url=url)
         jobs_json = filter_jobs_on_mode(response_json["data"], test_jobs, real_jobs)
 
         # API get jobs pagination exhaustion is indicated by empty next page (no last page flag)
-        while len(response_json["data"]) > 0:
+        logging.getLogger("up42.utils").setLevel(logging.CRITICAL)
```

Comment:
This together with line 190 mutes the logger so that not each page query gives another logger message